src/Component/OpenApi3/Generator/RequestBodyGenerator.php

```diff
@@ -123,6 +123,8 @@ public function getSerializeStatements(?RequestBody $requestBody, string $refere
 
             if (isset($this->generators[$contentType])) {
                 $generator = $this->generators[$contentType];
+            } elseif (str_ends_with($contentType, '+json')) {
+                $generator = $this->generators['application/json'];
             }
```

Comment:
I would prefer we do some catching from the generators const `JSON_TYPES`, like could we use regexes instead of static strings ? So we can match any string ending with `+json` and avoid this specific configuration ?